Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -622,6 +622,11 @@ extension PlaygroundController {
     func confirmHandler(_ paymentMethod: STPPaymentMethod,
                         _ shouldSavePaymentMethod: Bool,
                         _ intentCreationCallback: @escaping (Result<String, Error>) -> Void) {
+        // Sanity check the payment method
+        if paymentMethod.type == .card {
+            assert(paymentMethod.card != nil)
+        }
```

Comment:
Note this assertion fired in the `testLinkInlineSignup_deferred` added below before the change.